packages/bruno-app/src/components/Sidebar/Collections/Collection/RemoveCollection/index.js

```diff
@@ -10,15 +10,16 @@ const RemoveCollection = ({ onClose, collection }) => {
   const onConfirm = () => {
     dispatch(removeCollection(collection.uid))
       .then(() => {
-        toast.success('Collection removed');
+        toast.success('Collection disconnected');
         onClose();
       })
-      .catch(() => toast.error('An error occurred while removing the collection'));
+      .catch(() => toast.error('An error occurred while disconnecting the collection'));
   };
 
   return (
     <Modal size="sm" title="Remove Collection" confirmText="Remove" handleConfirm={onConfirm} handleCancel={onClose}>
-      Are you sure you want to remove collection <span className="font-semibold">{collection.name}</span> ?
+      Are you sure you want to disconnect collection <span className="font-semibold">{collection.name}</span> from
+      Bruno? It will remain in your file system anyways.
```

Comment:
I think the last sentence sounds weird with the "anyways". How about: "The collection files will remain in your file system in [Collection Path here]"